docs/deploy/finops-hub-0.4-privateendpoint.ui.json

```diff
@@ -0,0 +1,141 @@
+        "location": {
+          "label": "Location",
+          "resourceTypes": ["Microsoft.DataFactory/factories", "Microsoft.KeyVault/vaults", "Microsoft.Storage/storageAccounts,Microsoft.Network/privateEndpoints"]
+        }
+      }
+    },
+    "resourceTypes": [
+      "Microsoft.DataFactory/factories",
+      "Microsoft.KeyVault/vaults",
+      "Microsoft.Storage/storageAccounts",
+      "Microsoft.ManagedIdentity/userAssignedIdentities",
+      "Microsoft.Resources/deploymentScripts",
+      "Microsoft.Network/privateEndpoints"
+    ],
+    "basics": [
+      {
+        "name": "hubName",
+        "type": "Microsoft.Common.TextBox",
+        "label": "Name",
+        "defaultValue": "finops-hub",
+        "toolTip": "Name of the hub. Used to ensure unique resource names.",
+        "constraints": {
+          "required": true,
+          "regex": "^[a-zA-Z0-9][a-zA-Z0-9\\-]{0,61}[a-zA-Z0-9]$",
+          "validationMessage": "Name must be between 3 and 63 characters long and can contain only lowercase letters, numbers, and hyphens. The first and last characters in the name must be alphanumeric."
+        },
+        "visible": true
+      },
+      {
+        "name": "storageSku",
+        "type": "Microsoft.Common.DropDown",
+        "label": "Storage redundancy",
+        "defaultValue": "Locally-redundant (LRS) - Lowest cost",
+        "toolTip": "The data in your storage account is always replicated to ensure durability and high availability. Choose a replication strategy that matches your durability requirements. [Learn more](https://go.microsoft.com/fwlink/?linkid=2163103)",
+        "constraints": {
+          "required": false,
+          "allowedValues": [
+            {
+              "label": "Locally-redundant (LRS) - Lowest cost",
+              "value": "Premium_LRS"
+            },
+            {
+              "label": "Zone-redundant (ZRS) - High availability",
+              "value": "Premium_ZRS"
+            }
+          ]
+        },
+        "visible": true
+      },
+      {
+        "name": "subnetResourceId",
+        "type": "Microsoft.Common.TextBox",
+        "label": "Subnet Resource Id for Private Endpoint deployment (Optional)",
+        "defaultValue": "",
+        "toolTip": "Add a subnet resource id to deploy a private endpoint for the hub. If you don't provide a subnet resource id, the hub will be deployed without a private endpoint.)",
+        "constraints": {
+          "required": false,
+          "regex": "^[a-zA-Z0-9][a-zA-Z0-9\\-]{0,61}[a-zA-Z0-9]$",
+          "validationMessage": "Name must be between 3 and 63 characters long and can contain only lowercase letters, numbers, and hyphens. The first and last characters in the name must be alphanumeric."
+        },
```

Comment:
The name of the subnet itself can be up to 80 characters.

Reply:
I see you fixed it. Thanks!